hedera-node/src/main/java/com/hedera/services/txns/crypto/CryptoCreateTransitionLogic.java

```diff
@@ -102,9 +103,14 @@ public void doStateTransition() {
 			long balance = op.getInitialBalance();
 
 			if (op.hasProxyAccountID()) {
-				final var proxyValidation = ledger.lookUpAccountId(op.getProxyAccountID()).response();
-				if (proxyValidation != OK) {
-					txnCtx.setStatus(proxyValidation);
+				final var proxyLookUp = ledger.lookUpAccountId(op.getProxyAccountID());
```

Comment:
need to add `INVALID_PROXY_ACCOUNT_ID`

Reply:
Handled below. Need to discuss the 2 step workflow.